src/typescript-node/devcontainer-template.json

```diff
@@ -11,6 +11,7 @@
             "type": "string",
             "description": "Node.js version (use -bookworm, -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
+				"22-bookworm",
```

Comment:
Also, let's set `22-bookworm` as default in L22

Reply:
done